Feature/ae 1728 kv

muep opened the pull request.

muep committed:
AE-1728 Rename Tietopyyntö to Tietopyyntö 2021

muep committed:
AE-1728 Mark Tietopyyntö 2021 as valid=false

This alone does not do much, but it is expected that front-end will
stop offering this option for new toimepiteet.

muep committed:
AE-1728 New käytönvalvonta document templates

hkorpi commented on etp-db/src/main/sql/migration/repeatable/vk_template/r-0-vk-toimenpidetype.sql:
```diff
@@ -1,12 +1,13 @@
-(4, 'Käskypäätös', 'Käskypäätös (sv)', 5),
-(5, 'Valvonnan lopetus', 'Valvonnan lopetus (sv)', 6)
+(0, 'Valvonnan aloitus', 'Valvonnan aloitus (sv)', 1, true),
+(1, 'Tietopyyntö 2021', 'Begäran om uppgifter 2021', 2, false),
```
jätä tämä toistaiseksi päälle

hkorpi replied:
tarkoitus on että uuden prosessin voi ottaa käyttöön milloin vain sen jälkeen kun tämä tuki on asennettu

muep replied:
Vaihdoin trueksi, muta jätin nuo lisäsarakkeet niin on helppo vaihtaa takaisin

muep committed:
AE-1728 Re-enable Tietopyyntö 2021

This allows the valvojat to autonomously decide when they wish to
switch to the new process.

muep committed:
AE-1728 Re-enable the old asiakirjapohjat
